src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/PCCRegion.kt

```diff
@@ -13,5 +13,5 @@ data class PCCRegion(
   @NotNull @Id val id: String,
   @NotNull val name: String,
   @NotNull @ManyToOne @JoinColumn(name = "nps_region_id")
-  val region: NPSRegion
+  val npsRegion: NPSRegion
```

Comment:
👏

Reply:
1. Agree - with pccRegionid (also George has raised this) I will add that in - we have a PCCRegionDTO which holds these 2 fields
2. I have assumed (maybe wrong to assume) that we will need to return PCC Regions.  Code in this PR does that in the service layer (although I know I didn't ask you to review that bit yet).  It determines if the NPSRegion is set, and if so, does a call back to the db to get all PCCRegions associated with it.  There might be a need JPA way to do this- but will check.

Reply:
👍 Good call, it would be good to figure out preloading the linked category entities (service category, NPS regions, PCC regions, etc.) otherwise we'll have a massive N+1 query problem.

Reply:
One other question @sldblog -
What should the value of maximumAge be if we have no upper limit?  null/99/-1 ?

Reply:
for me, semantically, `null` would mean "unset", which I'd interpret as "no limit"
it's a bit harder to implement on client-side but the explicit value is worth it imo